src/main/java/jeryl/fyp/commons/core/Messages.java

```diff
@@ -7,7 +7,7 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_STUDENT_DISPLAYED_INDEX = "The person index provided is invalid";
```

Comment:
Should this be 'student index'?

Reply:
You mean to just put as `MESSAGE_INVALID_STUDENT_INDEX`?

Reply:
As in the string itself still uses 'person index'

Reply:
Ah! Silly me. Thanks for that! Fixed.